server/upstream/upstream-tarballs.txt

```diff
@@ -646,6 +646,7 @@ stk:httpls:https://ccrma.stanford.edu/software/stk/download.html
 sunpinyin:google:sunpinyin
 swfdec-mozilla:subdirhttpls:http://swfdec.freedesktop.org/download/swfdec-mozilla/
 swfdec:subdirhttpls:http://swfdec.freedesktop.org/download/swfdec/
+swig:sf:swig
```

Comment:
Unfortunately, that doesn't work. You can test this with: `./download-upstream-versions --debug "swig:sf:swig"`

It might be that the `sf` method is broken nowadays, and `download-upstream-versions` would need fixing...